### DISCUSSION TITLE
Is it possible to make the animations in blockbench?

### DISCUSSION THREAD

**Discussion by electricsteve** [KosmX/emotes#325] [Q&A]
So, I want to make a animation in blockbench, export it and use it. Is that possible or do I need to use blender?

**Comment by KosmX** [comment]
Yes, it is possible to use Blockbench.
But you can't animate items or bend from Blockbench.

Use this model:
https://github.com/KosmX/emotes/blob/dev/blend%20and%20other%20files/emote_creator.blend
